restricted-list.json

```diff
@@ -1521,7 +1521,27 @@
             "17157",
             "17158",
             "17159",
-            "17160"
+            "17160",
```

Comment:
this suggests that the FH cards are banned only for the Conclave 2.0 RL. if i'm not mistaken then you would also want to back-fill the `bannedCards` lists for previous RLs, like the FFG era ones.

Reply:
You are absolutely right. We did the same for the bin pack cards and i´ve pushed the changes you requested